doc/src/base/collections.md

```diff
@@ -235,6 +236,49 @@ Partially implemented by:
   * [`Iterators.Pairs`](@ref)
 
 ## Set-Like Collections
+A [`Set`](@ref) is a collection of elements with no duplicated elements and keys/indices.
+
+So in contrast to `Arrays`, in a `Set` you can have only one of each element, and the order of elements isn't important (whereas an array can have multiple copies of an element and their order is remembered). As a result, iterating over `Set` elements is slower than `Arrays`.
```

Comment:
I think the part about iterating being slower can be removed. There are `AbstractArray`s have the same properties as `Array` but are slower to iterate then `Set`.

Reply:
This line is also quite long.

Reply:
See also https://github.com/JuliaLang/julia/pull/33936#discussion_r373215632